<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);




</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>461</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: #2e2e2e;       /* Dark background color */
    /*border: 1px solid #4d4d4d;       /* Subtle border for separation */
    border-radius: 7px;              /* Rounded corners */
    padding: 10px;
    margin-top: 15px;                /* Space for the title */
    color: #cccccc;                  /* Lighter text color */


    subcontrol-origin: margin;
    padding: 0 3px;
    color: #b3b3b3;                  /* Lighter title color */
    font-weight: bold;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(154, 153, 150);font-size:17pt;</string>
    </property>
    <property name="text">
     <string>Login or crate an account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_field">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    
background-color: rgb(42, 42, 42);


border-top:none;  /* Pink border with full opacity */
border-left: none;
border-right: none;
border-bottom: 2px solid rgba(255, 105, 180, 1); ;
padding: 4px;
</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Email Address</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_field">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
    
background-color: rgb(42, 42, 42);
border-top:none;  /* Pink border with full opacity */
border-left: none;
border-right: none;
border-bottom: 2px solid rgba(255, 105, 180, 1); ;
padding: 4px;
</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #4d4d4d;</string>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::AddressBookNew"/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(28, 113, 216)</string>
    </property>
    <property name="text">
     <string>Forget Password ? </string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 2px solid #4d4d4d;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::MailForward"/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>461</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 240, 164);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../cda71e2a/Thumbnails-40.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="view_password">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>335</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(129, 61, 156);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>email_field</zorder>
   <zorder>password_field</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>login_button</zorder>
   <zorder>label</zorder>
   <zorder>view_password</zorder>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>691</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/newPrefix/233945db/wallpaper-wave-Dark.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/newPrefix/233945db/wallpaper-wave-Dark.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>590</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>EN</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DialogInformation"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WeatherClearNight"/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
